spotty/spotty/Scenes/Core/MusicScene/MusicViewController.swift

```diff
@@ -8,10 +8,165 @@
 import UIKit
 
 class MusicViewController: UIViewController {
-
+    
+    private var collectionView: UICollectionView!
+    private var dataSource: UICollectionViewDiffableDataSource<Section, AnyHashable>?
+    private let sections: [Section] = [.featuredPlaylists, .recentlyPlayedTracks, .recentlyPlayedArtists]
+    private let viewModel = MusicViewModel()
```

Comment:
As we discussed, let's refactor in this PR as _private var viewModel: MusicViewModelProtocol!_ and set it in _static functions create(....)_.

Reply:
done, without create()